lib/db.php

```diff
@@ -95,5 +95,26 @@ public static function Fetch($rs)
 			return $retVal;
 		}
 
+        /**
+         * Do bulk insert/update using transaction for faster processing
+         * @param $transactions
+         */
+		public static function BulkQuery($transactions)

```

Comment:
I've been meaning to write this one up for quite some time now. Thanks for adding!